packages/webpack/src/utils/postcss.ts

```diff
@@ -38,28 +36,25 @@ export async function getPostcssConfig (nuxt: Nuxt) {
     sourceMap: nuxt.options.webpack.cssSourceMap,
   })
 
+  const jiti = createJiti(nuxt.options.rootDir, {
+    interopDefault: true,
+    alias: nuxt.options.alias,
+  })
+
   // Keep the order of default plugins
   if (!Array.isArray(postcssOptions.plugins) && isPureObject(postcssOptions.plugins)) {
     // Map postcss plugins into instances on object mode once
-    const cwd = fileURLToPath(new URL('.', import.meta.url))
     const plugins: Plugin[] = []
     for (const pluginName of sortPlugins(postcssOptions)) {
       const pluginOptions = postcssOptions.plugins[pluginName]
       if (!pluginOptions) { continue }
 
-      const path = await tryResolveModule(pluginName, nuxt.options.modulesDir)
-
-      let pluginFn: (opts: Record<string, any>) => Plugin
-      // TODO: use jiti v2
-      if (path) {
-        pluginFn = await import(pathToFileURL(path).href).then(interopDefault)
-      } else {
-        console.warn(`[nuxt] could not import postcss plugin \`${pluginName}\` with ESM. Please report this as a bug.`)
-        // fall back to cjs
-        pluginFn = requireModule(pluginName, { paths: [cwd] })
-      }
+      const path = jiti.esmResolve(pluginName)
+      const pluginFn = (await jiti.import(path)) as (opts: Record<string, any>) => Plugin
```

Comment:
jiti will do esmResolve i think in module.ts only reason was that we need to also reuse same `path` in resolved meta